packages/freesewing.lab/hooks/useApp.js

```diff
@@ -1,67 +1,63 @@
-import { useState } from 'react'
+import { useState, useEffect } from 'react'
 import set from 'lodash.set'
-import mustache from 'mustache'
 // Stores state in local storage
 import useLocalStorage from 'shared/hooks/useLocalStorage.js'
 // Patterns
 import patterns from 'site/patterns.json'
-// Locales
-import { strings } from 'pkgs/i18n'
+// Locale and translation
+import { useRouter } from 'next/router'
+import { useTranslation } from 'next-i18next'
+import { capitalize } from 'shared/utils'
 
 // Initial navigation
-const initialNavigation = {
-  accessories: {
-    __title: 'accessoryPatterns',
-    __order: 'accessoryPatterns',
-    __linktitle: 'accessoryPatterns',
-    __slug: 'accessories',
-  },
-  blocks: {
-    __title: 'blockPatterns',
-    __order: 'blockPatterns',
-    __linktitle: 'blockPatterns',
-    __slug: 'blocks',
-  },
-  garments: {
-    __title: 'garmentPatterns',
-    __order: 'garmentPatterns',
-    __linktitle: 'GarmentPatterns',
-    __slug: 'garments',
-  },
-  utilities: {
-    __title: 'utilityPatterns',
-    __order: 'utilityPatterns',
-    __linktitle: 'utilityPatterns',
-    __slug: 'utilities',
-  },
-}
-for (const type in patterns) {
-  for (const design of patterns[type]) {
-    initialNavigation[type][design] = {
-      __title: design,
-      __order: design,
-      __linktitle: design,
-      __slug: `${type}/${design}`
-    }
+const initialNavigation = (locale, t) => {
+  const base = {
+    accessories: {
+      __title: t('accessoryPatterns'),
+      __order: t('accessoryPatterns'),
+      __linktitle: t('accessoryPatterns'),
```

Comment:
- `title`: The page title
- `linktitle`: The page title as used in menus and breadcrumbs. This is most useful for blogposts with titles like [We're JAMstack, we're JAMstack, we're JAMstack, we're JAMstack, we're JAMstack, we're JAMstack, we're JAMstack, and I hope you like JAMstack too](https://freesewing.org/blog/freesewing-goes-jamstack/) as including such a title in a menu is always going to look bad
- `order`: Controls the (navigation) menu order. By default, the order is alphabetic, but this can be overrules by setting an explicit order either in markdown frontmatter (for markdown) or in the navigation structure. Important catch here is that the default alphabetical order should be applied after translation (hence why it's wrapped in a translation call)
- `__` prefix: This is used to indicate this holds meta info about the page inside the navigation structure. Eg:

```js
{
  docs: {
    __title: 'Documentation',
    __linktitle: 'Documentation',
    __order: 'Documentation',
    sewing: {
      __title: 'Documentation',
      __linktitle: 'Documentation',
      __order: 'Documentation',
      },
   }
}
```    

There's a `docs` page and a `docs/sewing` page but `docs/__title` is not a page.

Reply:
I was just wondering if we could do with just `__title`.

By the way you may resolve such comments of mine. 
I only try to point out what caught my eye.

Reply:
I appreciate it, because it's good to ask **why** because if I don't have a good answer, it's something we should look into. So keep asking 😉

I tend to not resolve questions like these when answering them because that sort of passive/aggressively implies the discussion is now closed.

But what if my answer is not sufficient? What if it's unclear (or plain wrong) or you have follow-up questions.

So I prefer to not resolve it until I get confirmation that it's indeed resolved, as it is now